libs/gi-localization/assets/locales/en/char_Furina.json

```diff
@@ -0,0 +1,12 @@
+{
+  "fanfarePoints": "Fanfare",
+  "memberMult_": "Salon Members Mult.",
+  "numOffer": "Number of characters who offered HP",
+  "c1Inc": "Max Fanfare Increase",
+  "c1Bonus": "Fanfare on Cast",
+  "a4MemberDmg": "Salon Member DMG Bonus",
+  "a4HealInterval": "Singer of Many Waters Healing Interval",
+  "c2Points": "Fanfare points Above the limit",
+  "c6Healing": "Ousia Continuous Healing",
```

Comment:
Why does this get a custom string for healing? There is already a continuous healing string, and I remember when Baizhu came out you advocated for using the generic strings more often

Reply:
Because the header doesn't indicate what state it comes from and Pneuma gets a conditional, so imo Ousia ought to be mentioned too.